Src/GoogleApis.Auth.DotNet4/GoogleApis.Auth.DotNet4.csproj

```diff
@@ -33,6 +33,9 @@
     <DocumentationFile>Google.Apis.Auth.PlatformServices.xml</DocumentationFile>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="BouncyCastle.Crypto">

```

Comment:
It isn't quite clear to me as I don't see the other nuget packages listed here. Lets touch base offline and I can make the change.